src/views/protected/case/Detail.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <DetailCase></DetailCase>
```

Comment:
Redundant usage. Everything in `DetailCase` component is to be moved inside this view.